Client/Frontend/History/HistoryViewController.swift

```diff
@@ -26,6 +26,7 @@ class HistoryViewController: UITableViewController, UrlViewController {
                     println("Err: \(data.statusMessage)")
                 } else {
                     self.history = data
+                    self.tableView.reloadData()
```

Comment:
I don't know if this has changed in iOS8 but it used to be so that you cannot call `reloadData()` from any thread other than the main thread. I don't know who ultimately calls this but you may want to verify that or add a check for the current thread + dispatch to main thread.